modules/olm-catalogsource.adoc

```diff
@@ -56,6 +59,9 @@ status:
 ----
 <.> Name for the `CatalogSource` object. This value is also used as part of the name for the related pod that is created in the requested namespace.
 <.> Namespace to create the catalog available. To make the catalog available cluster-wide in all namespaces, set this value to `{global_ns}`. The default Red Hat-provided catalog sources also use the `{global_ns}` namespace. Otherwise, set the value to a specific namespace to make the Operator only available in that namespace.
+<.> Optional: To avoid cluster upgrades potentially leaving Operator installations in an unsupported state or without a continued update path, you can enable automatically changing your Operator catalog's index image version as part of cluster upgrades.
++
+Set the `olm.catalogImageTemplate` annotation to your index image name and use one or more of the Kubernetes cluster version variables as shown when constructing the template for the image tag. See the "Image template for custom catalog sources" section for more details.
```

Comment:
Should we link to the anchor of that section?

Reply:
It's in the "Additional resources" list at the assembly-level, but can't xref to it here since it's within the module, sadly.